src/tests/ftest/util/mpio_test_base.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python
 '''
-    (C) Copyright 2020 Intel Corporation.
+    (C) Copyright 2020-2021 Intel Corporation.
 
     Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This license should also be replaced with "SPDX-License-Identifier: BSD-2-Clause-Patent", right?

Reply:
Will handle these two files in next PR #5 for daos-6338.